packages/dropdown/src/react/__specs__/storyshots.spec.ts

```diff
@@ -3,10 +3,20 @@ import initStoryshots, {
   snapshotWithOptions
 } from '@storybook/addon-storyshots'
 
-const createNodeMock = () => document.createElement('div')
+const createNodeMock = () => {
+  document.createElement('div')
+}
 
 initStoryshots({
   configPath: path.resolve(__dirname, '../../../.storybook'),
   framework: 'react',
   test: snapshotWithOptions({ createNodeMock })
 })
+
+beforeEach(() => {
+  jest.spyOn(global.Math, 'random').mockReturnValue(0.123456789)
+})
+
+afterEach(() => {
+  jest.spyOn(global.Math, 'random').mockRestore()
+})
```

Comment:
if you're having testing issues with this mock - you can also mock out the util+hook

[example](https://github.com/pluralsight/design-system/blob/master/packages/field/src/react/__specs__/storyshots.spec.ts#L6)

Reply:
It works locally but this might solve the CI bug. Thanks!